src/script/lua_api/l_client.cpp

```diff
@@ -230,7 +230,11 @@ int ModApiClient::l_get_node_or_nil(lua_State *L)
 
 int ModApiClient::l_get_language(lua_State *L)
 {
+#ifdef WIN32
```

Comment:
the correct define is `_WIN32`

Reply:
Both WIN32 and _WIN32 are scattered throughout the codebase. They are also both recognised under gcc/glibc and visual studio. I'm not sure about other compilers. If _WIN32 is really preferred, perhaps it should be standardized?